static/css/inbox.less

```diff
@@ -1249,6 +1250,10 @@ a.fa:hover {
       }
     }
     .reset-filter {
+      padding-left: 10px;
+    }
+    .sort-results,
+    #sortDropdown {
```

Comment:
If you don't float them both they don't go all the way over. A smarter / more interested person than me could work out if we can change this while also retaining all the various emergent properties around how filters are laid out.

Reply:
Removing the float was part of the instructions yeah: now it's supposed to always align to the left.

Reply:
re: 1 & 2, I'm happy to change it, I was just following the other filters there, that have ids that are camel case.